GeneratedRegexPatterns/UnclassifiedPotentialSecurityKeys.json

```diff
@@ -34,10 +34,7 @@
     "Pattern": "(?i)authorization:(\\s|%20)bearer(\\s|%20)(?<refine>[0-9a-z][abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890_~.\\-+\\/=]*)([^abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890_~.\\-+/=]|$)",
     "Id": "SEC101/061",
     "Name": "OAuth2BearerToken",
-    "Signatures": [
-      "sig=",
-      "ret="
-    ],
+    "Signatures": null,
```

Comment:
>null [](http://example.com/codeflow?start=18&length=4)

it doesn't make sense to have no signature here if the regex itself is encoding things like `authorization` and `bearer`. either one would be ok, the choice would depend on what we think would be less likely to occur in other data. this format, oauth2 is a real bear due to the many variations in the way the secret is expressed and variance in the literal formats of the embedded secrets. blah.

Reply:
Hmm, can I make the signature case-insenstive?

Reply:
[https://www.rfc-editor.org/rfc/rfc6750#section-2.1](https://www.rfc-editor.org/rfc/rfc6750#section-2.1)

Reply:
[https://auth0.com/blog/the-bearer-token-case/](https://auth0.com/blog/the-bearer-token-case/)

Reply:
OK, so the RFC is ABNF and that string literal is case-insensitive, it appears, but there are suggestions that in the wild, there is a tendency to require the `Bearer` casing.

We could add `Bearer` and `bearer` as signatures and probably be ok.

Reply:
very good, but the question is whether case insensitivity is required in this sniff.

i am changing my suggestion, which may still be rejected, use `earer` as a sniff to cover the all lowercase and Pascal-Cased forms.

this rule is not in super great shape in any case, by the way, we could use a pass over it. all these other rules don't have a sniff either, so we could simply use null as they do. if this rule were to be improved to a point where the confidence was higher, the need for a sniff would become more important.

Reply:
> all these other rules don't have a sniff either, so we could simply use null as they do

This seems the most sensible to me.

> we could use a pass over it. 

I could file a follow-up on this.

Reply:
#150

Reply:
Can I get a stamp?